app/controllers/application_controller.rb

```diff
@@ -145,38 +146,10 @@ def most_focus(data_type, profile_id)
 
     data.empty? ? {} : top_three_focus(data)
   end
-  # rubocop:enable MethodLength
-
-  # rubocop:disable MethodLength
-  def tag_usage(data_id, data_type, user_id)
-    result = []
-    moments = user_moments(user_id).order('created_at DESC')
-    if data_type == 'category'
-      strategies = user_strategies(user_id).order('created_at DESC')
-      [moments, strategies].each do |records|
-        objs = []
-        records.find_each do |r|
-          objs.push(r) if data_included?(data_type, data_id, r)
-        end
-        result << objs
-      end
-    elsif data_type.in?(%w[mood strategy])
-      moments.find_each do |m|
-        result << m if data_included?(data_type, data_id, m)
-      end
-    end
-    result
-  end
-  # rubocop:enable MethodLength
+  # rubocop:enable MethodLength  
```

Comment:
Very minor, but it looks like there's trailing whitespace on this line, can you remove it?